docs/faqs-troubleshooting/faqs.md

```diff
@@ -0,0 +1,40 @@
+### Are mods or resource packs required?
+No mods are required, but our custom resource pack is mandatory for the best experience. Please enable our resource pack when you join the server.
+
+### Can I claim land to protect my builds?
+Yes, Theatria has a land-claim system to protect your builds. Learn how to use it [here](../gameplay-features//land-claiming/README.md).
+
+---
+
+## Community and Rules
+
+### What are the server rules?
+Our server rules are designed to create a fun and inclusive environment for everyone. You can view the full list of rules [here](../rules-policies/rules.md).
+
+---
+
+## Technical Questions
+
+### What versions of Minecraft are supported?
+We support most modern versions of Minecraft. For a detailed list of supported versions, visit [this page](https://docs.playtheatria.com/versions).
+
+### I’m experiencing lag. What can I do?
+Lag can happen for various reasons. Here are some troubleshooting tips:
+- Ensure your internet connection is stable.
```

Comment:
we can expand on these suggestions at an upcoming time, this is a good idea though so far.